Model/LdapUser.php

```diff
@@ -0,0 +1,316 @@
+<?php
```

Comment:
If this bundle have to ship a User object then there is no reason for copy the model from FOSUserBundle with attributes not related with LDAP (canonical versions, password, salt, ...)